docs/pymc_aesara.ipynb

```diff
@@ -0,0 +1,1819 @@
+{
```

Comment:
<p>This is nothing else *than*?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='79'/>

Reply:
<p>yes! Thx!</p>